thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -431,7 +462,32 @@ func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
 		// are deleted.
 		ContinueOnEmptyResult: true,
 		Selector:              r.Selector,
+		Exclusion:             r.Exclusion,
 	}
 
 	return nil
 }
+
+// parses annotation and label based selectors and exclusion from the command line input
+func (r *EvalFnRunner) parseSelectors() {
+	for _, s := range r.selectorAnno {
+		parts := strings.Split(s, "=")
+		key, value := parts[0], parts[1]
+		r.Selector.Annotations = map[string]string{key: value}
```

Comment:
As per the implementation, `r.Selector.Annotations` will end up with last `key=val`. Is that what we want here ?

I see some code duplication in this function, would be good if we can dedupe it.

Reply:
I noticed this at the same time you did, just fixed it

Reply:
done